Assginment2/app/src/main/java/com/htk/assginment2/presentation/HomeActivity.kt

```diff
@@ -0,0 +1,62 @@
+package com.htk.assginment2.presentation
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import com.htk.assginment2.R
+import com.htk.assginment2.databinding.ActivityHomeBinding
+
+class
+HomeActivity : AppCompatActivity() {
+    private lateinit var binding : ActivityHomeBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val homeFragment = HomeFragment()
+
+        val transaction = supportFragmentManager.beginTransaction()
+        transaction.add(R.id.home_fragment, homeFragment)
+        transaction.commit()
+        binding.tvHomename.setText("hotaek Kwak")
+        binding.tvHomeid.text = intent.getStringExtra("id")
+        binding.tvHomeintroduce.setText("안드 사랑해 보고싶다. 안드 사랑 해보고싶다.")
```

Comment:
```suggestion
        binding.tvHomename.text = "hotaek Kwak"
        binding.tvHomeid.text = intent.getStringExtra("id")
        binding.tvHomeintroduce.text = "안드 사랑해 보고싶다. 안드 사랑 해보고싶다."
```

굳이 setText함수를 쓰지않구 이렇게 작성하셔두 상관없으실 것 같아요..!(기왕 코틀린 쓰는 거 장점 활용 팍팍하면 좋으니까용 :))